index.html

```diff
@@ -1,172 +1,311 @@
-            <h2 class="about__title">About <br> Myself</h2>
-            <p class="about__description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <button class="btn btn-ligth" type="button">LET’S CONNECT</button>
-            <ul class="about__social">
-                <li class="about__link">
-                    <a href="#"><img src="./images/github.svg" alt="my github profile URL"></a>
-                </li>
-                <li class="about__link">
-                    <a href="#"><img src="./images/Linkedin.svg" alt="my Linkedin profile URL"></a>
-                </li>
-                <li class="about__link">
-                    <a href="#"><img src="./images/angellist.svg" alt="my angellist profile URL"></a>
-                </li>
-                <li class="about__link">
-                    <a href="#"><img src="./images/twitter.svg" alt="my twitter profile URL"></a>
-                </li>
-                <li class="about__link">
-                    <a href="#"><img src="./images/medium.svg" alt="my medium profile URL"></a>
-                </li>
-            </ul>
-            <button class="btn btn-primary" type="button">Get my resume</button>
-            <ul class="about__categories">
-                <li class="menu__link">
-                    <a href="#">Languages<img src="./images/IconDown.svg" alt=""></a>
-                    <ul class="about__categories-submenu">
-                        <li>
-                            <a class="submenu__link" href="#"><img src="./images/javascript.svg" alt=""> JavaScript</a>
-                        </li>
-                        <li>
-                            <a class="submenu__link" href="#"><img src="./images/html.svg" alt=""> HTML</a>
-                        </li>
-                        <li>
-                            <a class="submenu__link" href="#"><img src="./images/css.svg" alt=""> CSS</a>
-                        </li>
-                    </ul>
-                </li>
-                <li class="menu__link">
-                    <a href="#">Frameworks<img src="./images/IconRight.svg" alt=""></a>
-                </li>
-                <li class="menu__link">
-                    <a href="#">Skills<img src="./images/IconRight.svg" alt=""></a>
-                </li>
-
-            </ul>
-        </section>
-        
+          </li>
+          <li class="menu__link">
+            <a href="#"
+              >Frameworks<img src="./images/IconRight.svg" alt=""
+            /></a>
+          </li>
+          <li class="menu__link">
+            <a href="#">Skills<img src="./images/IconRight.svg" alt="" /></a>
+          </li>
+        </ul>
+      </section>
     </main>
-</body>
-</html>
\ No newline at end of file
+    <footer class="footer">
+      <fieldset>
+        <form
```

Comment:
- [ ] The semantically meaningful ordering of elements in the footer section, can be something like:
```
<footer>
    <section class="contact-me">
        <div>
            <h2> Contact Me </>
            <p> If you have an application you... </p>
        <div>
        <form action= >
        </form>
    </section>
    
    // here you can add other section such as foot navigation links, etc later on.
</footer>
```